pgdb/v1/insert.go

```diff
@@ -62,3 +62,45 @@ func Insert(msg DBReflectMessage) (string, []any, error) {
 
 	return q.ToSQL()
 }
+
+func BackfillPKSKV2(msg DBReflectMessage) (string, []any, error) {
+	dbr := msg.DBReflect()
+	desc := dbr.Descriptor()
+	tableName := desc.TableName()
+	record, err := dbr.Record()
+	if err != nil {
+		return "", nil, err
+	}
+
+	pkskv2Field := desc.PKSKV2Field()
+	pk := record["pb$pk"]
+	sk := record["pb$sk"]
+	record[pkskv2Field.Name] = fmt.Sprintf("%s|%s", pk.(string), sk.(string))
+
+	qb := goqu.Dialect("postgres")
+	q := qb.Insert(tableName).Prepared(true).Rows(record)
```

Comment:
can't this whole thing be an Update.... where?  Insert/OnConflict is because we're trying to get Upsert behavoirs

Reply:
seems like we don't need `upsert` since this is literally a backfill